openstudio-gems.gemspec

```diff
@@ -18,10 +18,11 @@ Gem::Specification.new do |spec|
   # gem version is specified in gemspec, gem source/location (e.g. github branch) can be specified in Gemfile
   # runtime dependency versions can be loosened while in development on branches if needed
   # runtime dependency versions should be specified as exact versions when merged to master or develop
-  spec.add_dependency 'openstudio-extension', '0.2.0'
-  spec.add_dependency 'openstudio-workflow', '2.0.0.pre1'
+  spec.add_dependency 'openstudio-extension', '0.2.1'
+  spec.add_dependency 'openstudio-workflow', '2.0.0'
   #spec.add_dependency 'openstudio-standards', '0.2.11-rc1'
-  spec.add_dependency 'openstudio_measure_tester', '0.2.0'
+  # Measure tester is redundant, it is included in openstudio-extension
+  #spec.add_dependency 'openstudio_measure_tester', '0.2.0'
```

Comment:
So is openstudio-workflow in that case. We should leave this here, because in each repo we use pessimistic version constrains, here we want FIXED version for the core gems.